src/features/settings/Settings.tsx

```diff
@@ -38,12 +47,25 @@ export function Settings() {
     []
   );
 
+  const handleUnifiedEtaChange = useCallback(
+    (_: FormEvent<HTMLDivElement>, data: { value: string | undefined }) => {
+      setUnifiedEta(data.value === "true");
+      dispatch(
+        changeSettings({
+          id: "unifiedEta",
+          name: "unifiedEta",
+          value: (data.value === "true").toString(),
```

Comment:
I wonder why we should store the string of boolean instead of a boolean value directly.

This also applies to all other `=== "true"`

Reply:
string is a unified storage type for all setting items, as does localstorage keys and values